desiredSHAs.csv

```diff
@@ -8,3 +8,4 @@ https://github.com/open-cluster-management/grc-chart,stable/grc,grc,release-2.4
 https://github.com/open-cluster-management/insights-chart,stable/insights-chart,policyreport,release-2.4
 https://github.com/open-cluster-management/management-ingress-chart,stable/management-ingress,management-ingress,release-2.4
 https://github.com/open-cluster-management/search-chart,stable/search-prod,search-prod,release-2.4
+https://github.com/open-cluster-management/cluster-backup-chart,stable/cluster-backup-chart,release-2.4
```

Comment:
Before the release-2.4 is should have the chart name, which is `cluster-backup-chart`. So it would be `https://github.com/open-cluster-management/cluster-backup-chart,stable/cluster-backup-chart,cluster-backup-chart,release-2.4`